configure.ac

```diff
@@ -153,12 +153,17 @@ fi
 AC_DEFINE_UNQUOTED(CONF_PATH_RANDOMDEV, "$opt_randomdev", [Random device path.])
 
 dnl Check for cracklib
+AC_ARG_ENABLE([cracklib],
+        AS_HELP_STRING([--disable-cracklib], [disable cracklib dictionary check]))
```

Comment:
As I've replied, you have a good point that it should be probably cracklib-check and --disable-cracklib-check.